src/assets/css/footer.scss

```
@@ -0,0 +1,206 @@
+//Prueba para tener contenido y ver el footer al final
+body > p {
+  background-color: $moramia-blue;
+  height: 100vw;
+}
+* {
+  box-sizing: border-box;
+  list-style: none;
+  margin: 0px;
+  padding: 0px;
+  text-decoration: none;
+}
+.footer {
+  background-color: $footer-background;
+  bottom: 0;
+  color: $moramia-white;
+  display: block;
+  left: 0;
+  overflow: hidden;
+  padding: 5em 0;
+  position: relative;
+  transition: all 0.4s ease;
+  width: 100vw;
+}
+.footer-content {
+  display: block;
+  font-family: $font-stack;
+  height: 100%;
+  margin: auto auto;
```

```suggestion
  margin: 0 auto;
```
top and bottom margins are not working here since you're setting height:100%

---

Done